punic.php

```diff
@@ -2,7 +2,7 @@
 
 spl_autoload_register(
     function ($class) {
-        if (strpos($class, 'Punic\\') !== 0) {
+        if (0 !== strpos($class, 'Punic\\')) {
```

Comment:
Like I wrote before, I don't like Yoda style.
I think you had to switch to it because of the change you made in the `composer.json` from `"friendsofphp/php-cs-fixer": "2.2.*"` to `"friendsofphp/php-cs-fixer": "^2.2"`... Am I correct?